### PR TITLE
Initialize project with rollup typescript

### PR DESCRIPTION
**Pull request by winChawakorn**
Initialize project with rollup typescript

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+export default {
+  input: 'src/index.ts',
+  output: [
+    {
+      file: pkg.main,
+      format: 'cjs'
+    },
+    {
+      file: pkg.module,
+      format: 'es' // the preferred format
```
Comment (babestvl) [inline rollup.config.js]: Delete code comment

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "esModuleInterop": true,
+    "plugins": [
+      {
+        "name": "typescript-tslint-plugin",
+        "configFile": "./tslint.json"
+      }
+    ],
+    "declaration": true
+  }
+}
```
Comment (babestvl) [inline tsconfig.json]: Add new line

File: package.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  "devDependencies": {
+    "rollup": "^1.21.2",
+    "rollup-plugin-includepaths": "^0.2.3",
+    "rollup-plugin-terser": "^5.1.2",
+    "rollup-plugin-typescript2": "^0.24.2",
+    "tslint": "^5.20.0",
+    "typescript": "^3.6.3",
+    "typescript-tslint-plugin": "^0.5.4"
+  }
+}
```
Comment (babestvl) [inline package.json]: Add new line

**Comment by winChawakorn** [comment]
#4 Set up rollup with typescript

**Pull request merged**